terraform/accounts/main/jenkins.tf

```diff
@@ -22,6 +22,20 @@ module "jenkins_2" {
   log_bucket_name                   = module.jenkins_elb_log_bucket.bucket_id
 }
 
+module "jenkins_3" {
+  source                            = "../../modules/jenkins/jenkins"
+  name                              = "jenkins3"
+  aws_account_and_jenkins_login_ips = var.aws_account_and_jenkins_login_ips
+  jenkins_public_key_name           = aws_key_pair.jenkins.key_name # Or ${var.ssh_key_name}
```

Comment:
`# Or ${var.ssh_key_name}` - what does this mean?

Reply:
@bjgill this is a copy-paste from the current instance block, but appears to be a reference that you may wish to use your own variable (as defined in https://github.com/alphagov/digitalmarketplace-aws/blob/889d2e369bbd25c939f69d8bec39aa6c4143938d/terraform/accounts/main/jenkins.tf#L2-L3)